llpc/test/shaderdb/OpLoad_StackBoundsCheck_struct.pipe

```diff
@@ -0,0 +1,137 @@
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: {{^// LLPC}} pipeline patching results
+
+; SHADERTEST: = icmp ult i32 %[[#]], -16
```

Comment:
Why -16?

Reply:
llvm rearranges the code that it skips the load if `int(color_in.y * 255) < -16`.

Reply:
I think this deserves a comment in code.